### PR TITLE
profiles: info_vars add more variables, sorting

### PR DESCRIPTION
**Pull request by Alessandro-Barbieri**
Signed-off-by: Alessandro Barbieri <lssndrbarbieri@gmail.com>

Bug: https://bugs.gentoo.org/722456
Closes: https://bugs.gentoo.org/780477

### CODE REVIEW

**Comment by Alessandro-Barbieri** [comment]
Ping

File: profiles/info_vars [hunk 1]
```diff
@@ -7,35 +7,69 @@
...
 CONFIG_PROTECT
 CONFIG_PROTECT_MASK
+CONFIG_SHELL
+CPP
 CPPFLAGS
 CTARGET
 CXXFLAGS
 DISTDIR
-FEATURES
+DWP
```
Comment (thesamesam) [inline profiles/info_vars:30]: What is this?

File: profiles/info_vars [hunk 2]
```diff
@@ -7,35 +7,69 @@
...
 ACCEPT_LICENSE
+ADDR2LINE
+AR
+ARFLAGS
+AS
+ASFLAGS
 CC
+CCLD
 CXX
+CXXFILT
```
Comment (thesamesam) [inline profiles/info_vars:18]: I'm not convinced this needs to be in every single emerge --info call.

File: profiles/info_vars [hunk 3]
```diff
@@ -7,35 +7,69 @@
 ##
 ACCEPT_KEYWORDS
 ACCEPT_LICENSE
+ADDR2LINE
+AR
+ARFLAGS
+AS
+ASFLAGS
 CC
+CCLD
```
Comment (thesamesam) [inline profiles/info_vars:16]: People are really typically setting this? ditto several others

File: profiles/info_vars (hunk 2 above)
Comment (thesamesam) [reply profiles/info_vars:18]: .. oh, these are only shown if set. I guess it's okay.

File: profiles/info_vars (hunk 1 above)
Comment (thesamesam) [reply profiles/info_vars:30]: I'll drop this as I have no idea what it is.

**Pull request closed without merging**